internal/provider/resource_uuid.go

```diff
@@ -148,11 +148,17 @@ func (r *uuidResource) ImportState(ctx context.Context, req resource.ImportState
 
 	var state uuidModelV0
 
-	state.ID.Value = result
-	state.Result.Value = result
-	state.Keepers.ElemType = types.StringType
+	keepers, diags := types.MapValue(types.StringType, nil)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	state.ID = types.StringValue(result)
+	state.Result = types.StringValue(result)
+	state.Keepers = keepers
```

Comment:
```suggestion
	state.ID = types.StringValue(result)
	state.Result = types.StringValue(result)
	state.Keepers = types.MapNull(types.StringType)
```

Reply:
Have replaced with `types.MapValueMust(types.StringType, nil)` as the terraform-plugin-framework `v0.14` implementation was using a known value for keepers.